### DISCUSSION TITLE
评分 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#447] [General]
# 评分 - Miao Yu | 于淼

https://yufree.cn/cn/2023/12/17/score/

**Comment by earfanfan** [comment]
哇嗷，世另我。怎么大家都在走如此相似的路，约莫就是接受自己、放下过去、面对自我诸如此类的。咱俩好像没撒共同点来着，随便想想，哈哈，我还真想到两个。其一，咱俩是生活在同一个时代的人，接受了同样的时代赋予的考验。其二，很久以前，咱俩讨论过确定性与不确定性，可以算作都对社会规训的世俗评价标准有过质疑，源头应该是不同，我对“标准”的质疑是看了庄子以及女性主义相关的书，你的质疑源头就你自己知了。

基于此，咱俩应该都遇到过一种相似的情况，举个例子，有小伙伴可能会置身于标准之内，寻求标准里偏向客观全面的一端，而我会质疑绝对的客观和全面是否存在，沉浸于个人追求的客观和全面，是不是就会把别人追求的客观和全面视作异端。

**Reply by yufree** [reply]
我没看明白你在说啥。另外我好像也没啥放不下的过去，我一直都挺接受自己的。至于说质疑，那属于职业病，不质疑还做啥研究。

**Reply by earfanfan** [reply]
原来我没说清楚啊，那我再捋捋。“接受自己、放下过去、面对自我”是功夫熊猫电影三部曲的主题，前两点对我来说是个大坑，爬出来很难，没想到对你来说居然都不算是个坑，看来属于你的大坑还在别处。你的博客题目是评分，从俺这个读者的角度看讲的是你脱离了大众评分标准，也就不会再被大众标准塑造你的涉猎范围和感受，我就以为你写的是面对自我。由于最近我与其他朋友交流，大家都有类似感受，质疑社会规训带来的世俗评价标准，我自然了解我和我的朋友们为撒会走这样的路，但是觉得咱俩没撒共同点，于是瞎分析了一下为撒你也会走这样的路。以本读者对你的了解，我感觉你看到这就会吐槽下好面吃好饭不就得了，想这些干嘛咧，哈哈。若是允许我继续瞎想的话，从我的角度看，天下苦秦久矣，大家慢慢醒悟到被某种东西无形操控着、被社会价值观念裹挟太久，于是会进入寻找自我认同或者认同自我的阶段。

至于一楼的第二段，假如以前的你是小三水，现在的你是大三水，在小三水长成大三水的过程中，会有许多社会评价标准经过你，如果小三水与小伙伴们都置身于同一个评价标准之内，大家就只是争个很绝对的输赢，比如考试啊之类的东西，或者做数据分析、科学研究是否能够达到绝对的客观和全面，但是大三水显然已经锤炼出来了独属于自己的评价标准（ps是否稳固未可知），于是乎小三水的小伙伴们若是仍然置身于原来的比较标准之内，就和现在的大三水分道扬镳了。这一段我也不知道我在写撒，我自己还迷糊着，而且我也看不懂你为何从“评分”的标准里跳出来了，但还是又跳进去 AI 的评价标准里了，说好的写作是休息呢，结果居然还是忍不住去追求更加优越了……

看样子，每个独立个体的绝对自我本来也是不存在的，“自我”这个概念也本来就是几个世纪前才发明出来的，看上去像是一个用来描述个体特性的概念，实际上好像也无法脱离开群体的某种不知道咋描述的共性吧。

**Reply by yufree** [reply]
我还是没看明白，用AI纯属调戏，反正我也不改。再说我只是觉得评分会降低多样性，也不至于完全否定评分，或者说我看问题一直没那么绝对，概念的对立可能没啥实际意义，基于概念的判断更多是空对空输出。如果说我真正反感什么，大概就是各种形而上的对空输出了，虽然我也会用。

**Reply by earfanfan** [reply]
我好像明白我在说撒了。是这样的，我前面絮叨的东西都有一个前提，那就是成长经历中有一个遭受思想控制的过程，倒也不是一个人在控制我，我也说不清那种无形力量到底是撒东西，但在我产生质疑之前，审美标准、道德标准、成为优秀的人的标准、活得好的标准等等都是直接灌到我脑子里的。在我开始质疑之后，对于审美、道德等各方面的评价标准开始偏离“社会评价标准”（ps我也说不清楚社会评价标准到底是撒）。

好了，我对“社会评价标准”的质疑还在进行中，又踩到对“自我”这个概念的质疑之中了，这个我也还迷糊得很，回头耕一篇博客再来继续。不过这些标准对我来说并不只是虚空的概念，比如上周末我只是在家看电影电视剧，虽然看的时候感觉非常放松有好好地享受周末，可是看完了以后会忍不住批判自己没有去做读书、写作、运动、社交等似乎是“自我提升”的事。

另，原来对我来说千难万难的问题对你来说居然不存在啊，难道外面的世界完全没有任何的社会文化氛围上的控制么，回头我得瞅瞅世界历史了，之前我莫名其妙地默认叫魂式动员全国的运动在全世界都可能存在过，也以为思想控制也在全世界都存在，我之前一点也没想到“思想控制”对你居然从不生效。话说前几年我刚来到你的博客瞎溜达的时候，曾经想过要是你是我的老师就好了，后来有一天做梦真得梦到你是我的老师了，可是居然是高中老师啊喂，哈哈，从此就不把你当老师看了。不过每次与你交流，总是能让我发现新大陆，就好像以前都是闭着眼睛看世界似的。

**Reply by yufree** [reply]
想多了，周末做什么那是你的自由，所谓自我提升更像是满足外界的期望，只是很多人希望有个外部动力来作为自己存在的意义，或者说有个“正确”的三观指导人生。于我而言，也有类似的东西，但我不太好区分哪些是我通过家庭、教育或文化复读周围人的影响，哪些又是自己的主见，或者单纯就是看了个广告被影响的，本能上我会抗拒外界强加的影响，但实际中确实也分不出来，所以从我这边能做的就是忽略这个本质上无意义的议题，去做我自以为不受影响的事。简单说，我明确知道别人眼中的我跟我自我感知的我一定是不一样的，同理，我眼中的别人也跟这个人眼中的自己是不一样的，进一步说，我以为的文化特征或评分更像是我自己竖的靶子，我看似对抗外界，其实对抗的还是过去的自己，也就是你所谓的跟自己和解，但再往深一步想，我压根就不想对抗哪来的和解，只要拆了或忽略自己的虚空靶子就可以了，没那么复杂。当然，为了谋生，有些对抗是必须的。

**Reply by earfanfan** [reply]
你这“想多了”三个字又莫名其妙戳中我的笑点了，哈哈，我决定再来拍拍你的马屁。讲真，本读者觉得与你的交流中从未感觉到你用你的标准来衡量我，而只是用你已经形成的三观来解释我吐槽的主题（PS这次我脑补的是你在帮助我），这让我感受到很充分的平等和尊重，如果将来某一天三水你真得去教书成为了一名老师，我想一定会是一名好老师的。

你的答案有几分道理，很实用。单论我自己的话，其实我也不知道“自我提升”的需求是怎么产生的，可以说是受到环境影响像是要满足外界期望，但也可以说是我本来就有一种强烈的追求优越的“优越情结”，是自己和自己比较才产生的。我觉得你的答案很好，不过对于现阶段的我来说，因为脑子里被灌入的一些糟粕观念带来的畸形标准还没被拔除干净，所以有些时候还是需要继续费点劲再做点区分。

说回评分，我有用豆瓣来标记想看和看过的电影电视剧和书籍，评分也是会参考的，榜单也看见过，不过很早就觉得豆瓣的评分和榜单都是有倾向的，这可能跟在那打分的人群有关，比如我喜欢看一些TVB的电视剧，但是豆瓣上大家就总是絮叨TVB布景粗糙而打低分，还有一些我觉得很好看值得8分的电影电视剧，大家总是因为一些我不知道咋描述的偏见而打低分，当然也有一些我觉得一般的电视剧，大家会因为一些价值观念上的东西而打高分，有一些电视剧的评分随时间推移人们的价值观念改变而改变。

至于书籍嘛，我就没参考过榜单，有些时候会被微信读书上别人划线的评论内容安利别的书，有些是无意中搜索到的，比如不记得哪天看到了周飞舟这个名字，搜索周飞舟会看到一个“三周”的称呼，于是我又搜到了周飞舟周雪光周黎安，再单独搜就会搜到他们三人的代表作品，读周飞舟的《以利为利》时看书里提到作者一些学友的名字又能搜到一些新的书目，再有就是每次看一本书也会看别人写的书评，有人会提到某些书的某些观点应该和其他书的其他观点对比来看……哈哈，有些高分书籍我每次看完都会专门搜一下差评来看。有些我觉得很好的书，发现此书在TOP250也会感到高兴，有些我觉得写得很好但是不在TOP250的也会感慨一下此书生不逢时啊打分的人数太少之类的。

嗷，我想到为撒你说“想多了”三个字会戳笑点了，哈哈，因为我默认你也认为思考本身自带乐趣属性，哈哈哈，乐意思考的时候就将此行为称为思考，想着想着脑壳打结虐到自己了就称此行为为想多了，总而言之就是怎样都是合乎自己的理就对了。

**Reply by yufree** [reply]
没必要合乎自己的理，想多了就是说已经到了钻牛角尖了，虚空讨论没有终点。我不需要逻辑自洽，没答案的问题放着或者不讨论就可以，坑多了也就不纠结了，总想着想清楚了再下一步是自欺欺人，很多问题大概率永远想不清楚，但也无所谓。

**Comment by zocozocozoco** [comment]
「后果就是越来越难享受前互联网时代偶然看到佳作的感觉了」问题是不是之前是没有那么多前置的所谓「标准」（也就是评分），所以感觉都是自己给自己的，且阈值不是那么高。但现在相当于有标准了，既然有标准，大众就愿意排名，阈值也就提高。反过来理论上讲，如果看的目标都是评分不怎么高的，是不是会出现「偶然看到佳作的感觉」。

这个新闻也可以描述下我说的：https://new.qq.com/rain/a/20231212A09VT800

大众点评也有类似的问题，和大众点评的产品经理聊，评分这个事情要从几个维度去看，一是时间，二是显示评分，三是有没有差评以及商家对差评的处理手段。他们通过线下实际的打分和线上评分的对比，得出4-4.5分，开业时间超过1年的店，实际上是相对好吃的店（从概率上讲），如果评分特别高，基本都是刷分，口味这个事儿，的确不一定是自己说了算。

说到量，文化积累的量是不是是普通人的一个必然经过的过程呢（天资异于常人的排除），于老师到这个过程，是不是也是已经过了附庸大众的阶段呢。经验主义和理性主义的对立有些简单化，我现在觉得评分其实也是组成自己的一部分，不太可能彻底摆脱这套体系，但是得分清楚哪些是基于这些衍生出来的，这部分可能才是「自己」。

**Reply by yufree** [reply]
有道理，评分这东西在统计上有意义，进行群体决策也有意义，但用整体评价指导个人决策可看作某种驯化或自我驯化。很像是贝叶斯统计里那个先验概率，个人会依附自己认可的似然度更新为后验概率，但我怀疑很多人只考虑先验概率了。当然，从商业角度，用量化数值驯化客户构建权威是建立品牌获取超额利润所必须的，但我会质疑这个数灵敏度如何？评分是否独立？是否有系统性偏差？这并不否认这个评分有价值，只是质疑对个人参考价值大小，毕竟先看评分后看作品就已经不独立了，更可能被光环效应或三人成虎所引导。

我必然经历了附庸大众的过程，而且互联网的评分加速了这个过程，更像是前互联网时代口口相传模式的高效率延伸。我一直也是大众的一部分，对于自己不了解的领域，有个参考分确实有助于起步阶段规避低效信息，但在我比较熟悉的领域里评分是很可笑的，因为很多规律不会因为评分低就不存在，也不会因为评分高就是真实的。说到底，评分是面向大众的参考，但不应该成为个人决策的制约。

**Reply by botsp** [reply]
受教了，两位的观点和讲述很有启发。属于"自我"的有多大程度上是外部世界塑造的、而有多少是自己在努力去构建的，也许这个过程就是我们奔跑的旅程吧。

即使没有一个"量化的评分"，自己也是时常与大众约定俗成的评价进行着缠斗，哈哈。

**Reply by zocozocozoco** [reply]
之前有个概念叫做「二手人生」，依靠评分一定程度上会使得回溯过去时强化这种感觉，说到底，还是「我」存在的意义问题。

**Comment by XiangyunHuang** [comment]
关于评分，我想起之前看电影的时候，想找高分电影，就把豆瓣排行榜抓了出来，之后按榜单刷那些还没看过的电影。大概十部八部之后，共同特点也很明显，同类题材更加。很快也就刷不下去了，之后，再看其它榜单外的电影更觉没意思，节凑快没新意没有深度。打个比方，好吃的菜多吃也吃不下去，不好吃的菜就提不起筷子。好吃的菜不常有，不好吃的菜天天有。不妨吃段时间的斋再去吃好吃的。我往往是自己吃完再看别人的评论，然后和自己的感觉比较，可能会获得不一样的视角。

**Reply by yufree** [reply]
电影最大特色在于用镜头引导叙事，打破这个限制观影体验差，服从这个限制会有不真实的感觉，就像全世界都围着主角在转，对导演要求很高，同时很多作品也挑观众，不是目标受众就会莫名其妙。本就是众口难调，能在大多数人那里得高分一定是一个大多数人都接受的交集，这东西好莱坞编剧烂熟于心。不过个人观影总是这类还是会疲劳，总是要有不一样的体验，哪怕这个体验很糟糕，否则就会被工业化电影产业驯化为爆米花观众了。

**Reply by XiangyunHuang** [reply]
有的电影评分高、评论数多，我觉得是很多年积累起来的，不一定是大多数人的认同和交集，比如《教父》，而又有很多爆米花电影出来后跟网红一样很快死亡了。所以，有的电影是艺术，有的电影是快餐配着爆米花。顺着时间捋下来，排行榜上同类的电影似乎总想着要突破，但实际上，很多其它电影都是为了生存，为了一部想拍的电影准备。我记得王晶有一次做客锵锵三人行，他说【你投资我拍，过个肥年，不拍会惹事】，要优先考虑政策、投资人等等，留给导演发挥的空间很小。黄渤有一次给李克强总理说希望多一点创作自由。我看过一点冯小刚的记录片，他的电影【1942】肯定是他很想要的，但不是市场需要的。

再说回来评分，我觉得评分是一些观众、投资人需要的，分别对应于省事省时间、宣传攒票房，谈论越多热度越高，流量越多票房越多。电影一旦剪辑完，除了宣传就没导演什么事了。所以，陈丹青弄《局部》说过这样一句话，电影视频也是剪辑的艺术。我觉得这就是不真实的感觉的来源。

**Reply by yufree** [reply]
西方文艺作品是有叙事模版的，《千面英雄》模版几乎能扫荡九成以上的电影，《教父》也不例外，电影产业到今天不到130年，还没有真正经过时间检验，今天看起来是艺术的东西更多还是因为有些主题放在当下有所共鸣，几十年后大家纠结的东西变了，经典的定义也会随之改变。当下市场不需要的以后可能会奉为经典，当下经典以后可能就是泛泛之作或者需要重新解释，中文中的经典是十三经，但历朝历代都会有人重新注疏，看似“翻译”古人的智慧，实际是借古人之口重新解释新事物，归拢回经典体系，最早经书里的东西很多早就没现实意义了。前几年美国有个讲全氟化合物的小众纪录片获奖无数，但因为我做过这方面研究，里面很多东西纯粹就是误导，只是无法否认这部纪录片确实给学术界带来了研究经费与公众关注度，符合普通人的认知水平，我猜这部片早晚也会成为高分小众的“经典”，但从内容上看全是硬伤。电影的工业化与本身的艺术性是否矛盾我没有定论，但现代编剧确实都是在用桥段拼凑的方法堆出经典，[这个网站](https://tvtropes.org/)里的内容你过一遍再随意看一部所谓的“经典”就会发现全是套路，当然这是纯技术视角，不妨碍作品本身还是比较出色。

延伸来说，今年好莱坞的编剧演员大罢工很可能是历史上最后一次罢工，AGI时代留给编剧演员讨价还价的空间越来越少。更有意思的地方在于，即使是AGI套路感满满的作品，也好于大多数普通编剧的水平，而培养一个优秀的编剧必然要从普通开始。工业化的套路与个性化定制化的艺术需求并不矛盾，在当前这个时代尤其如此，评分更像是无关痛痒的标注，AI早就知道你想要什么了。至于未来，大概率会出现那种你输入主题、情绪、演员名（可能需要买许可）等要素就能给你虚拟生成一部专属电影的服务，完美符合你对艺术的所有想象，对个人而言绝对是满分之作且千人千面，此时我们这个时代的导演又会倔强的说些什么呢？

**Reply by XiangyunHuang** [reply]
> 至于未来，大概率会出现那种你输入主题、情绪、演员名（可能需要买许可）等要素就能给你虚拟生成一部专属电影的服务，完美符合你对艺术的所有想象，对个人而言绝对是满分之作且千人千面，此时我们这个时代的导演又会倔强的说些什么呢？

恰好我之前看到姜文在锵锵三人行上回答这个问题，他说（大意），如果人人都能这么自定义电影，想要什么就能出什么，有那么多选择，他就自己做导演了。而实际上，又有多少人可以不受拘束地自由创作呢？有多少人具备这样的创作条件，即使算上 AGI 。

姜文还说，如果要千人千面，就需要大量的不同的剧本，还需要导演配合剧本运用镜头，最后，还是繁荣了内容创作者，比如编剧、导演等。我想达到一定规模后，不同的剧本和镜头组合起来就千人千面了。

**Reply by earfanfan** [reply]
我也想对三水关于 AI 的发言吐吐槽，我觉得三水想象的这种人类输入要素，AI 生成专属电影的形式未必能够完美符合人类的需求。输入要素这步就有坑，暂且假装输入是输入语言文字，语言文字这个工具载体的有限、用户自身表达能力的有限都有可能使得 AI 未必能够百分百理解人类的需求。继续假装这个形式和现实中用户和开发者沟通需求细节一样，反复多次总能让 AI 理解用户要什么，用户验证结果是否符合要求的时间成本也降下来了，但是电影是造梦的艺术，有时候看电影就是看一些自己一个人想象不出来的东西，如果用户自己都提不出来完整的需求又怎么让 AI 去想象和创造呢？确实，电影里的很多要素都可以归纳总结出来一些很标准的技术，比如剧本的套路、配乐的套路，把一部电影拆分成很多不同的细节，使得抽象变得具体，复杂变得简单，把细节组合成整体的过程也可以大量复制，但就是这个组合的过程我总觉得 AI 还欠缺一些我不知道咋描述的东西。

**Reply by yufree** [reply]
输入的确实是语言文字，但隐藏的输入是你的电子档案，例如网页浏览记录、网购记录、通讯记录、病历、出勤记录等，作为你的个性化AI，大概率比你更清楚你真实需要什么或者不知道什么，此时生成一个你感到惊艳的电影并不困难，例如AI知道你没见过昙花一现，电影里特意给一个特写满足你好奇心，其实你的需求早就已经隐含在你的日常数据里了，让你输入的提示词只是一个提取的契机。
另，为啥你们都是工作时间回贴？

**Reply by XiangyunHuang** [reply]
> 输入的确实是语言文字，但隐藏的输入是你的电子档案，例如网页浏览记录、网购记录、通讯记录、病历、出勤记录等，作为你的个性化AI，大概率比你更清楚你真实需要什么或者不知道什么，此时**生成一个你感到惊艳的电影**并不困难

可能不同的人会有不同的感受。于我而言，这样的电影不会让我感到惊艳而是**惊吓**。我曾经遇到一个一点也不熟的人，但是和他没聊几句，他就知道我都快忘记了我的往事（比如 10 年前看过什么书，在某平台上说过什么话），后来，我发现，这人会在网络上各个地方搜集/查找我的信息，当时，我就心理感到害怕和抗拒，从此，落下一点病根，手机都上了防窥膜，因为我还不得不继续和他接触。如果说电影制作真的利用上了我散布在网上的信息，可以，但是，不要让我察觉，不要直给。

**Reply by zocozocozoco** [reply]
>  我也想对三水关于 AI 的发言吐吐槽，我觉得三水想象的这种人类输入要素，AI 生成专属电影的形式未必能够完美符合人类的需求。输入要素这步就有坑，暂且假装输入是输入语言文字，语言文字这个工具载体的有限、用户自身表达能力的有限都有可能使得 AI 未必能够百分百理解人类的需求。继续假装这个形式和现实中用户和开发者沟通需求细节一样，反复多次总能让 AI 理解用户要什么，用户验证结果是否符合要求的时间成本也降下来了，但是电影是造梦的艺术，有时候看电影就是看一些自己一个人想象不出来的东西，如果用户自己都提不出来完整的需求又怎么让 AI 去想象和创造呢？确实，电影里的很多要素都可以归纳总结出来一些很标准的技术，比如剧本的套路、配乐的套路，把一部电影拆分成很多不同的细节，使得抽象变得具体，复杂变得简单，把细节组合成整体的过程也可以大量复制，但就是这个组合的过程我总觉得 AI 还欠缺一些我不知道咋描述的东西。

对「符合人类的需求」存疑，我理解这个需求是文化需求，先定义什么是人类的文化需求，时间维度上，一千年前人类的文化需求和现在的文化需求基本是不一样的，如果给一千年前的人类，给一个如今AI生成的剧本展示出来，未必不是惊天之举。所以AGI生成的是否符合人类需求，可能只是时间和样本，和匹配程度的问题。

**Reply by earfanfan** [reply]
>另，为啥你们都是工作时间回贴？

对于我，是因为三水你都是在你的早上我的晚上睡觉时间点回帖啊，等到了白天，按我不大可能憋得住等到晚上回去再吐槽，所以就在工作间隙回帖。你前面说叫我别钻牛角尖，放着不交流也可以，我能忍住不回复已经很了不起了。

又另，湘云讲的这个事这个人真得太恐怖了，恐怖到我看到楼上讲定义什么是人类的文化需求以后脑子都转不动了。

**Reply by XiangyunHuang** [reply]
> 另，为啥你们都是工作时间回贴？

在机场等飞机呀，休假呀，你不需要及时回复的。

隐私、安全和质量不是 AGI 能解决的问题，必须结合人一起来把关。有的代码自动生成工具，不会提供所谓的隐私、安全和质量保障，生命周期维护等等，自动生成电影也会面临类似的问题。

**Reply by earfanfan** [reply]
关于 AI 自动生成电影来满足人类需求，我又产生了两点疑问。其一，如果必须有“输入”才算下达指令的话，那么在人类输入需求的时候，就已经了解了电影内容了啊，剧透影响观影体验，我还记得当年看到那位天才导演今敏的几部动画电影时，真得感到超级震惊。当然 AI 也可以猜。其二，我也说不清看电影的口味是咋形成的，有时候是看了某一部电影后才对某一种特点的风格感兴趣，有时候是特意找本来偏爱的或者本来感兴趣的题材来看，我们几个讨论这件事的时候，本身都有了不少观影的积累，可是如果有一天真得全世界大量播放的都是 AI 电影，那么年龄小的小孩本身也还没积累数据，如果 AI 总是需要历史数据来预测未来，可是我感觉看到喜欢的电影这件事有时候是被生活中的随即事件触发的，在 AI 和小孩的情况下，如果缺少不确定性因素的话，不又回到信息茧房的问题上了么。

另外，三水提到古代经书原来的内容从现在看没撒实际意义，也确实如你所言是当代的解读对当代的人才有意义。我昨天刚好看黄仁宇的《万历十五年》里有一章提到类似观点，明朝中叶资本主义萌芽，贫富差距加大的情况下人们会倾向于放弃固守的“仁义道德”而追求实际利益，正因为群众思想松动了，统治者急需一种新的思想让群众思想归位，这时朱氏根据儒家经典新注释解读出来的理学就“上位”了。但电影跟古文经典有点不一样，电影是画面，每个人都可以直接用眼睛看用耳朵听，而那些古文却需要经由看得懂古文的人解读后再传递给普通人，经典古文哲学论述偏多，而电影更像是描摹不同的生活不同的世界，说不定在久远的未来，电影更具有考古价值。
